src/consensus/versionbits.cpp

```diff
@@ -0,0 +1,394 @@
+// Copyright (c) 2009-2015 Eric Lombrozo, The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+/*
+                     _             _     _ _                         
+ __   _____ _ __ ___(_) ___  _ __ | |__ (_) |_ ___   ___ _ __  _ __  
```

Comment:
Doxygen supports [documentation on a file](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdfile) level, which seems more useful than ascii arts.

> One for the initial author and initial year and a second one for the bitcoin core devs.

Is this really necessary? There is already the much more precise git history, and to my understanding, all contributors qualify as "Bitcoin Core developers".